pypeman/plugins/remoteadmin/views.py

```diff
@@ -80,7 +80,9 @@ async def stop_channel(request, ws=None):
 
 async def list_msgs(request, ws=None):
     """
-    List first `count` messages from message store of specified channel.
+    List first `count` message infos from message store of specified channel.
```

Comment:
perhaps we can document differently. perhaps it was even me who wrote that documentation, but it doesn't make sense to write
first `count` message infos if `count` is not a parameter of the function.

where does count from.


Other question:

did you really chek every olace that was using this func to see whether the new behaviour doesn't break anythin?